site/content/in-dev/unreleased/access-control.md

```diff
@@ -123,7 +123,7 @@ To grant the full set of privileges (drop, list, read, write, etc.) on an object
 | TABLE_READ_DATA | Enables reading data from the table by receiving short-lived read-only storage credentials from the catalog. |
 | TABLE_WRITE_DATA | Enables writing data to the table by receiving short-lived read+write storage credentials from the catalog. |
 | TABLE_FULL_METADATA | Grants all table privileges, except TABLE_READ_DATA and TABLE_WRITE_DATA, which need to be granted individually. |
-| TABLE_ATTACH_POLICY | Enables attaching policy to a table. Applying policy at a table will override the same policy type defined at namespace and catalog |
+| TABLE_ATTACH_POLICY | Enables attaching policy to a table. Applying a policy at a table will override the same policy type defined at namespace and catalog |
```

Comment:
I don't think we need to mention the inheritance mechanism here, also it’s also not accurate, since a policy might not be inheritable. 
```suggestion
| TABLE_ATTACH_POLICY | Enables attaching policy to a table. |
```